skbio/stats/composition.py

```diff
@@ -2058,3 +2060,489 @@ def dirmult_ttest(
         "Reject null hypothesis",
     ]
     return res[col_order]
+
+
+def _type_cast_to_float(df):
+    """Attempt to cast all of the values in dataframe to float.
+
+    This will try to type cast all of the series within the
+    dataframe into floats.  If a column cannot be type casted,
+    it will be kept as is.
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+
+    Returns
+    -------
+    pd.DataFrame
+    """
+    # Implementation based on https://github.com/mortonjt/differential/blob/65752567ef4cf303471405b0a9be503eb10a0bbb/differential/util.py#L4
+    # TODO: Will need to improve this, as this is a very hacky solution.
+    for c in df.columns:
+        s = df[c]
+        try:
+            df[c] = s.astype(np.float64)
+        except Exception:
+            continue
+    return df
+
+
+def _lme_call(
+    formula,
+    metadata,
+    table,
+    groups,
+    reml=True,
```

Comment:
Same here, let's see if we can exclude `reml` to see if users can still pass it in via `fit_kwargs`.

Looking over this more carefully, it may require `fit_kwargs = {'reml': True}`

Reply:
`reml` is set to True by default in `statsmodels` functions, so users wouldn't necessarily need to pass it as `fit_kwargs = {'reml': True}`. But I did test it out and the functionality remains intact.